It seems we have a new version of GCC (15) on Ubuntu 25.10 Questing Quokka which is throwing maybe uninitialized errors for Boost Ublas.

GCC version
```
g++ (Ubuntu 15.2.0-4ubuntu4) 15.2.0
Copyright (C) 2025 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
```

Ublas error
```
/__w/Chaste/Chaste/mesh/src/common/AbstractMesh.cpp: In member function ‘ChasteCuboid<SPACE_DIM> AbstractMesh<ELEMENT_DIM, SPACE_DIM>::CalculateBoundingBox(const std::vector<Node<SPACE_DIM>*>&) const [with unsigned int ELEMENT_DIM = 1; unsigned int SPACE_DIM = 1]’:
/__w/Chaste/Chaste/mesh/src/common/AbstractMesh.cpp:251:45: error: ‘*&position.boost::numeric::ublas::c_vector<double, 1>::data_[0]’ may be used uninitialized [-Werror=maybe-uninitialized]
  251 |                 c_vector<double, SPACE_DIM> position = rNodes[index]->rGetLocation();
      |                                             ^~~~~~~~
```

This might need measures similar to those implemented in #231

--- a/cmake/Modules/ChasteCompilerFlags.cmake
+++ b/cmake/Modules/ChasteCompilerFlags.cmake
@@ -60,7 +60,7 @@ elseif (${CMAKE_CXX_COMPILER_ID} STREQUAL "GNU")
     # uninitialised c_vectors in Release mode. See https://github.com/Chaste/Chaste/issues/231 for full details.
     if (CMAKE_BUILD_TYPE MATCHES "Release|RelWithDebInfo|MinSizeRel"
             AND CMAKE_CXX_COMPILER_VERSION VERSION_GREATER_EQUAL 9.4
-            AND CMAKE_CXX_COMPILER_VERSION VERSION_LESS 15)
+            AND CMAKE_CXX_COMPILER_VERSION VERSION_LESS 16)
 
         if (CMAKE_CXX_COMPILER_VERSION VERSION_LESS 11.1)
             set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-maybe-uninitialized -Wno-array-bounds -Wno-stringop-overflow")

